avocado/core/runners/exec_test.py

```diff
@@ -34,6 +34,9 @@ class ExecTestRunner(BaseRunner):
     name = 'exec-test'
     description = 'Runner for standalone executables treated as tests'
 
+    CONFIGURATION_USED = ['run.test_parameters',
```

Comment:
Hi @clebergnu I just find out that the `run.test_parameters` is not used in `exec-test` or I didn't find the usage, but I found that `run.keep_tmp` is used.

Reply:
Good catch! Fixing it.